contracts/src/7683/BasicSwap7683.sol

```diff
@@ -372,6 +373,11 @@ abstract contract BasicSwap7683 is Base7683 {
         if (_orderId != OrderEncoder.id(orderData)) revert InvalidOrderId();
         if (block.timestamp > orderData.fillDeadline) revert OrderFillExpired();
         if (orderData.destinationDomain != _localDomain()) revert InvalidOrderDomain();
+        // Enforce auction winner
```

Comment:
My comment is somewhat outside the scope of this PR, but:
the `fill` method has a `_fillerData` param that is passed to this method `_fillOrder` but not used. We are however setting it on `filledOrders`, which is the param we use for settlements via xChainRead, i.e. whatever address the solver passes into `_fillerData` will be the address that receives the user's escrow on the source chain. I think this is a good feature since it lets solvers set a different address to receive funds, but we should be more explicit about it by:
1. removing the param from `_fillOrder`
2. changing the name to something like `depositRecipient`

no need to address in this PR though

Reply:
1. Yes lot's of grooming to do it those contracts !
2. fillerData name is actually from the erc, and it's meant to be arbitrary data (for eg. that's where we would put the sig for winner enforcement with auction after intent open) so I think we should rather describe the format and fields expected in those bytes in comments